apps/docs/src/guide/types/raw-slice.md

```diff
@@ -2,14 +2,12 @@
 
 A dynamic array of values can be represented using the `RawSlice` type. A raw slice can be a value reference or a raw pointer.
 
-<<< @/../../docs-snippets/src/guide/types/raw-slice.test.ts#raw-slice-1{ts:line-numbers}
-
```

Comment:
Should this have been removed?

(Please resolve as confirmation)

Reply:
Same as https://github.com/FuelLabs/fuels-ts/pull/3340#discussion_r1808780178